### PR TITLE
build: new icon set gulp process

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

- Added a script to convert from strokes to fills using https://docs.oslllo.com/svg-fixer/master/#/getting-started/introduction
- The script flattens internal <path>s into single <path>

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: svg-stroke-to-fill.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const path = require('path');
+const SVGFixer = require('oslllo-svg-fixer');
```
Comment (josedialpad) [inline svg-stroke-to-fill.js]: nit: what does "SVGFixer" mean?  Checking the library, I think we could use `svgStrokeToFill` that's what it does.

File: package.json [hunk 2]
```diff
@@ -71,6 +73,7 @@
     "markdownlint": "^0.26.2",
     "markdownlint-cli": "^0.32.2",
     "npm-run-all": "^4.1.5",
+    "oslllo-svg-fixer": "^2.2.0",
```
Comment (josedialpad) [inline package.json:75]: Do we use this package in replacement of https://www.npmjs.com/package/svg-outline-stroke?

File: package.json [hunk 3]
```diff
@@ -19,7 +19,9 @@
     "prepublishOnly": "npm run build",
     "release": "semantic-release --no-ci --extends ./release-local.config.js",
     "start": "gulp watch",
-    "stylelint": "npx stylelint 'lib/build/less/**/*.{less,css}'"
+    "stylelint": "npx stylelint 'lib/build/less/**/*.{less,css}'",
+    "build:icons": "gulp icons",
+    "build:icons-parallel": "gulp iconsSeriesAsync"
```
Comment (josedialpad) [inline package.json]:
```suggestion
    "build:icons-parallel": "gulp iconsParallel"
```

File: gulpfile.js [hunk 4]
```diff
@@ -100,6 +117,8 @@ const paths = {
     brandInput: './lib/build/svg/brand/**/*.svg',
     brandOutputLib: './lib/dist/svg/brand/',
     outputVue: './lib/dist/vue/icons/',
+    newInputRoot: './lib/build/svg/new',
+    newOutputRoot: './lib/dist/svg/new',
```
Comment (josedialpad) [inline gulpfile.js:122]: Can we use v7 prefix and directory instead of new?

File: package.json (hunk 2 above)
Comment (juliodialpad) [reply package.json:75]: yes, svg-outline-stroke didn't work as expected

File: gulpfile.js (hunk 4 above)
Comment (juliodialpad) [reply gulpfile.js:122]: This naming is just temporary, I need to ask if the older icons are going to be removed.

File: svg-stroke-to-fill.js (hunk 1 above)
Comment (juliodialpad) [reply svg-stroke-to-fill.js]: Sounds good, I'll rename it

**Review by braddialpad — APPROVED** [review]

**Comment by juliodialpad** [comment]
Lowered the resolution so the script takes only like 5 seconds, later will need to compare the icons until we find the right "resolution-performance" balance.

**Review by braddialpad — APPROVED** [review]

**Pull request merged**